### PR TITLE
plum vision UI changes

### PR DESCRIPTION
**Pull request by Arjita-01**
Plum vision page -
1. created dropdown for fixed allocation and position
2. based on the selected option show the table and input fields
3. Integrated the upload portfolio api

### CODE REVIEW

File: src/layout/SubSidebar/ChartSubSidebar/ChartCanvas.jsx [hunk 1]
```diff
@@ -171,7 +171,8 @@ const ChartCanvas = ({ setCodeValue, setGraphNm }) => {
                                                     zeroline: true,
                                                     visible: true,
                                                 },
-                                                height: 300,
+                                                height: 400,
+                                                width: 1000,
```
Comment (punit-fm) [inline src/layout/SubSidebar/ChartSubSidebar/ChartCanvas.jsx:175]: fixing the width will make this unresponsive

File: src/pages/PlumVision/PlumVision.jsx [hunk 2]
```diff
@@ -54,28 +60,63 @@ export default function PlumVision() {
...
 			setIsFileUploaded(false);
 		}
+		event.target.value = null;
 	}
 
 	const handlePlayButtonClick = () => {
-		if (isFileUploaded && inputRange.startDate && inputRange.endDate) {
-			localStorage.setItem(LOCALSTORAGE.DATETIME, JSON.stringify(inputRange));
-			navigate('/plum-dashboard');
+		if (!isFileUploaded) {
```
Comment (punit-fm) [inline src/pages/PlumVision/PlumVision.jsx:67]: use error handling in all functions to avoid unexpected crashes

**Review by punit-fm — APPROVED** [review]

File: src/layout/SubSidebar/ChartSubSidebar/ChartCanvas.jsx (hunk 1 above)
Comment (Arjita-01) [reply src/layout/SubSidebar/ChartSubSidebar/ChartCanvas.jsx:175]: Ok removed it and handled the width through style. I will push it in next PR.

File: src/pages/PlumVision/PlumVision.jsx (hunk 2 above)
Comment (Arjita-01) [reply src/pages/PlumVision/PlumVision.jsx:67]: Ok I will fix it .

**Pull request merged**